spring-cloud-aws-dynamodb/src/main/java/io/awspring/cloud/dynamodb/DynamoDbAdminOperations.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.dynamodb;
+
+import software.amazon.awssdk.enhanced.dynamodb.model.DescribeTableEnhancedResponse;
+
+public interface DynamoDbAdminOperations {
```

Comment:
considering that it's super rare to create or drop tables in runtime (tables are created usually with CloudFormation, CDK or TF) i am not convinced we should have this interface.